jsx/FilterForm.js

```diff
@@ -5,7 +5,8 @@
  * @version 1.1.0
  *
  */
-
+import React, {Component} from 'react';
+import PropTypes from 'prop-types';
```

Comment:
I understand that you are importing React because you've removed 'React.' but do PropTypes still also need to be imported?

Reply:
@zaliqarosli Yes since it's no longer React.PropTyptes